src/main/java/de/fraunhofer/aisec/codyze/analysis/Finding.java

```diff
@@ -183,4 +177,14 @@ public void prettyPrintShort(@NonNull PrintStream out) {
 		out.println(lines + ": " + (isProblem ? "(BAD)  " : "(GOOD) ") + shortMsg + ": " + logMsg);
 	}
 
+	public String toShortMessage() {
+		String lines;
+		if (locations.size() == 1) {
+			lines = (locations.get(0).getRegion().getStartLine() + 1) + "";
+		} else {
+			lines = "[" + locations.stream().map(loc -> "" + (loc.getRegion().getStartLine() + 1)).sorted().distinct().collect(Collectors.joining(", ")) + "]";
```

Comment:
maybe use `prettyPrintShort`? it has the same code.

Reply:
🤔 we may need to sort this one out. All these output methods do similar stuff. One problem with `prettyPrintShort` is that it requires a `PrintStream` and prints the string directly to some output 'file'.

I would postpone the redesign for now.